test/tools/unified-spec-runner/entities.ts

```diff
@@ -149,13 +179,23 @@ export class UnifiedMongoClient extends MongoClient {
   } as const;
 
   constructor(uri: string, description: ClientEntity) {
+    const logCollector = new UnifiedLogCollector();
     super(uri, {
       monitorCommands: true,
       [Symbol.for('@@mdb.skipPingOnConnect')]: true,
+      [Symbol.for('@@mdb.enableMongoLogger')]: true,
+      [Symbol.for('@@mdb.internalLoggerConfig')]: {
+        componentSeverities: description.observeLogMessages,
+        logDestination: logCollector
+      },
```

Comment:
Passing in the logDestination like this comes with the bonus that TS doesn't complain about options that don't exist